src/Stratis.Bitcoin.Features.Miner/Staking/PosMinting.cs

```diff
@@ -925,44 +935,29 @@ private bool SignTransactionInput(UtxoStakeDescription input, Transaction transa
         }
 
         /// <inheritdoc/>
-        public async Task<(Money balance, Money immature)> GetMatureBalanceAsync(List<UtxoStakeDescription> utxoStakeDescriptions)
+        public async Task<Money> GetMatureBalanceAsync(List<UtxoStakeDescription> utxoStakeDescriptions)
```

Comment:
Not really related to this PR but we can make this method private and remove it from the interface as it is actually not used anywhere else. Perhaps now we can actually remove this altogether?

Reply:
Ah hang on @fassadlr this might be used by the SqlWallet repo no? if we want to be able to stake with the new wallet.